src/js/views/globals/search/patient-search_views.js

```diff
@@ -56,18 +56,30 @@ const EmptyView = View.extend({
 
 const PatientSearchPicklist = Picklist.extend({
   className: 'patient-search__picklist',
-  getItemSearchText() {
-    return `${ this.model.get('first_name') } ${ this.model.get('last_name') }`;
-  },
   itemClassName: 'patient-search__picklist-item',
   itemTemplate: hbs`
-    {{matchText text search}}{{~ remove_whitespace ~}}
-    <span class="patient-search__picklist-item-meta">{{formatDateTime birth_date "MM/DD/YYYY"}}</span>
+    <div class="patient-search__picklist-item-name u-text--overflow">
+      <span>{{matchText name search}}{{~ remove_whitespace ~}}</span>
+    </div>
+    <div class="patient-search__picklist-item-meta">
+      <div class="patient-search__picklist-item-dob u-text--overflow">
+        {{formatDateTime birth_date "MM/DD/YYYY"}}
+      </div>
+      {{#if identifier}}
+        <div class="patient-search__picklist-item-identifier u-text--overflow">
+          {{matchText identifier search}}
```

Comment:
I think this could be `{{matchText identifiers[0].value search}}`.. not 100% confident..

Reply:
That would work much better. Handlebars doesn't like it though.

![Screen Shot 2022-09-13 at 12 02 01 PM 1](https://user-images.githubusercontent.com/35355575/189962859-fabc3a61-f7bd-428f-aff5-2449392606b0.png)

Reply:
try ``{matchText identifiers.0.value search}}`

Reply:
Nice, that works.